frontend/tests/Utils.php

```diff
@@ -433,6 +433,30 @@ public static function runAggregateFeedback(): void {
         );
     }
 
+    public static function runCheckPlagiarisms(): void {
+        self::commit();
+        $host_arg = '';
+        $host_chunks = explode(':', OMEGAUP_DB_HOST, 2);
+        if (count($host_chunks) == 2) {
+            [$hostname, $port] = $host_chunks;
+            $host_arg .= ' --host ' . escapeshellarg($hostname);
+            $host_arg .= ' --port ' . escapeshellarg($port);
+        } else {
+            [$hostname] = $host_chunks;
+            $host_arg .= ' --host ' . escapeshellarg($hostname);
+        }
+        self::shellExec(
+            ('python3 ' .
+             dirname(__DIR__, 2) . '/stuff/cron/plagiarism_detector.py' .
+             ' --verbose ' .
+             ' --logfile ' . escapeshellarg(OMEGAUP_LOG_FILE) .
+             $host_arg .
+             ' --user ' . escapeshellarg(OMEGAUP_DB_USER) .
+             ' --database ' . escapeshellarg(OMEGAUP_DB_NAME) .
+             ' --password ' . escapeshellarg(OMEGAUP_DB_PASS))
```

Comment:
```suggestion
             ' --password ' . escapeshellarg(OMEGAUP_DB_PASS)) .
             ' --local-downloader-dir ' . escapeshellarg(OMEGAUP_TEST_ROOT)
```

see the comments in the Python file for an explanation of this.

Reply:
Understood and Done.